topics/transcriptomics/tutorials/droplet-quantification-preprocessing/tutorial.md

```diff
@@ -0,0 +1,767 @@
+>    - *"Field to place first in output table"*: `transcript_id` (This is accessing the column you identified above!)
+>    - *"Suppress header line in output?"*: `Yes` (The next tool (Alevin) does not expect a header)
+>    - *"Comma-separated list of field names to extract from the GTF (default: use all fields)"*: `transcript_id,gene_id` (This calls the first column to be the transcript_id, and the second the gene_id. Thus, your key can turn transcripts into genes)
+>    - *"Append version to transcript identifiers?"*: `Yes` (The Ensembl FASTA files usually have these, and since we need the FASTA transcriptome and the GTF gene information to work together, we need to append these!)
+>    - *"Flag mitochondrial features?"*: `No`
+>    - *"Filter a FASTA-format cDNA file to match annotations?"*: `Yes`
+>    - {% icon param-file %} *"FASTA-format cDNA/transcript file"*: `FASTA file in your history` {% icon galaxy-history %}
+>    - *"Annotation field to match with sequences"*: `transcript_id`
+> 2. Rename {% icon galaxy-pencil %} the annotation table to `Map`
+>
+> 3. Rename {% icon galaxy-pencil %} the uncompressed filtered FASTA file to `Filtered FASTA`
+{: .hands_on}
+
+## 1.3 Generate a transcriptome index & quantify!
+
+Alevin collapses the steps involved in dealing with dscRNA-seq into a single process. Such tools need to compare the sequences in your sample to a reference containing all the likely transcript sequences (a 'transcriptome'). This will contain the biological transcript sequences known for a given species, and perhaps also technical sequences such as 'spike ins' if you have those.
+
+> ### {% icon details %} How does Alevin work?
+>
+> To be able to search a transcriptome quickly, Alevin needs to convert the text (FASTA) format sequences into something it can search quickly, called an 'index'. The index is in a binary rather than human-readable format, but allows fast lookup by Alevin. Because the types of biological and technical sequences we need to include in the index can vary between experiments, and because we often want to use the most up-to-date reference sequences from Ensembl or NCBI, we can end up re-making the indices quite often. Making these indices is time-consuming! Have a look at the uncompressed FASTA to see what it starts with.
+>
+{: .details}
+
+We now have:
+
+* Barcode/ UMI reads
+* cDNA reads
+* transcript/ gene mapping
+* filtered FASTA
+
+We can now run Alevin. In some public instances, Alevin won't show up if you search for it. Instead, you have to click the Single Cell tab at the left and scroll down to the Alevin tool. Tip: If you click the tools from the tutorial option within Galaxy, you'll always have the correct version of the tool! In this case, it is: (Galaxy Version 0.14.1.2+galaxy1) - it should be default. If not, click 'Versions' and choose that version.   
+
+![Clicking the tool](../../images/wab-tutorial-in-galaxy.png "Tutorial option at the top right in Galaxy")
```

Comment:
These images look out of place- are they just placeholders?

Reply:
The second one. Can't really make that image til the tutorial is in!